.codeclimate.yml

```diff
@@ -0,0 +1,16 @@
+version: "2"
+plugins:
+  rubocop:
+    enabled: true
+exclude_patterns:
+- "config/"
+- "db/"
+- "dist/"
```

Comment:
Do we need this exclude pattern? Shouldn't it reuse the ones defined by rubocop?

Reply:
This comment is actually not about line 8, but the entire block. Sorry, I misclicked :(

Reply:
I am not sure. 
The reason I added it is because CodeClimate uses the rubocop.yml only for the rubocop check, obviously, but it is doing [other checks as well](https://docs.codeclimate.com/v1.0/docs/default-analysis-configuration#section-maintainability-checks).
And we want to keep those, right? So, with the default settings copied, the behaviour of the Code Quality check as we have it now, should stay the same.

Reply:
As I understand it, we explicitly only enable rubocop. There are more, yes, and we can consider adding them in the future. Looking at the docs you linked, I see that the exclude pattern is already the default one for CC so I would prefer to remove them to be in sync with what CC thinks best. But I don't have hard feelings about this.

Reply:
I'll remove them. If we miss them, we'll bring them back.